addons/l10n_cl_website_sale/static/src/js/fields/l10n_cl_fields.js

```diff
@@ -0,0 +1,76 @@
+/** @odoo-module **/
+import { WebsiteSale } from 'website_sale.website_sale';
+
+WebsiteSale.include({
+    events: Object.assign(WebsiteSale.prototype.events, {
+        'change #div_l10n_cl_sii_taxpayer_type': '_onChangeClickAddTypeDocumentCl',
+    }),
+    /**
+     * @override
+     */
+    start: function () {
+        var def = this._super.apply(this, arguments);
+        var vat = document.querySelector('input[name=vat]')
+        var radio_invoice = document.getElementById('radio_invoice')
+        if (document.getElementById('radio_ticket') && this.target.baseURI.indexOf('/shop/address')){
+            if(vat.value){
+                sessionStorage.setItem('partner_vat', vat.value)
+            }
+            if ( !radio_invoice || !radio_invoice.checked ){
+                this._hiddenFieldCl()
+                vat.value = ''
+            }
+        }
+        return def;
+    },
+
+    /**
+     * Event click, hidden fields l10n_cl_activity_description if l10n_cl_sii_taxpayer_type is 'ticket'
+     * @private
+     */
+    _onChangeClickAddTypeDocumentCl: function(ev) {
+        var vat = document.querySelector('input[name=vat]')
+        if ( ev.target.value == 3 ) {
+            this._hiddenFieldCl()
+            sessionStorage.setItem('partner_vat', vat.value)
+            vat.value = ''
+        } else {
+            this._showFieldCl()
+            vat.value = sessionStorage.getItem('partner_vat')
+            sessionStorage.removeItem('partner_vat')
+        }
+    },
+
+    _onChangeCountry: function (ev) {
+        this._super(...arguments)
+        this._validateCountryCl()
+    },
+
+    _validateCountryCl: function () {
+        var radio_invoice = document.getElementById('radio_invoice')
+        if(radio_invoice){
+            var country = document.getElementById('country_id')
+            country = country.item(country.selectedIndex)
+            if(country.textContent.trim().toLowerCase() != 'chile' && country.value != '' && country){
```

Comment:
And what if the website is in another language?  (Like FR/NL: Chili)

Reply:
Hi, I made the necessary changes, the new logic is based on the country code as this is not affected by language.